docs/guide/dev/integration/react/v2/README.md

```diff
@@ -0,0 +1,526 @@
+| ------------- | --------------------- | ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| config        | object                | `{}`                                     | Config object passed to editor's constructor.<br> See config details [here](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html).                                                                                                                                                                                                                                             |
+| debug         | bool                  | `undefined`                              | Toggles debugging mode. Use it to log info related to editor events.                                                                                                                                                                                                                                                                                                                          |
+| dispatchEvent | func                  | `undefined`                              | Dispatches editor events.<br> Pass a custom function or `dispatch` from `useReducer`. Event type and payload will be passed as part of the first argument: `({ type, payload }) => { ... }`. Event name is provided as `type` and its value is prefixed with `__CKE__` in order to facilitate integration with `useReducer`. Use `CKEditorEventAction` helper to access prefixed event names. |
+| editorUrl     | string                | link to [CDN](https://cdn.ckeditor.com/) | See [Customizing CKEditor Preset or Version](#customizing-ckeditor-preset-or-version).                                                                                                                                                                                                                                                                                                        |
+| element       | object                | `undefined`                              | Required prop. Ref to underlying to DOM element.                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Let's try to avoid abbreviations.
```suggestion
| element       | object                | `undefined`                              | Required. Reference to underlying to DOM element.                                                                                                                                                                                                                                                                                                                                              |
```